.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,15 +8,22 @@ assignees: ''
 ---
 
 **Describe the bug**
+<!-- This form is used to report software defects such as inaccurate/erratic behavior, crashes, etc. If you have general questions about using the software or are encountering difficulties with installation, please create a topic in the forum (see the "Discussions" tab at the top of this page). -->
```

Comment:
I'm worried the nudge to go to the Discussions page might be too subtle. Especially because, if someone is thinking of creating an issue for a non-bug question, they'll arrive [here](https://github.com/neuropoly/axondeepseg/issues/new/choose) first:

![Screenshot from 2020-12-19 09-56-16](https://user-images.githubusercontent.com/16181459/102692340-ae703100-41e0-11eb-9d0b-ba9e5cd4db2b.png)

They might just choose "Open a blank issue" and never even see this little note here about Discussions.

What do you think about creating a third (dummy) issue template called "Everything else" or "Questions and concerns" or something like that, then in the description, include a link to the Discussions page? Basically to say "don't open a blank issue, go here instead", to fill the gaps in GitHub's UI design. :wink:

Reply:
hum... in general i'm reluctant to offer ppl [too many choices](https://www.ted.com/talks/barry_schwartz_the_paradox_of_choice?language=en). I already find that 2 is too much. Also, too much text scares ppl. I would personally go with something like: 
- one button (bug/feature request)
- when ppl click there, there is one big message that says: if you are a user seeking help, please use the discussion forum instead --> URL)

Reply:
Regarding @mathieuboudreau's suggestion in Slack: 

> But I wouldn’t overthink the issues template too much; let’s just get something simple out there and see if it’s effective

I think I agree, and so maybe disregard my suggestion. _(One vs. two vs. three templates seems like something to check back in on, once we've seen how people treat the choices in practice. :stuck_out_tongue_closed_eyes:)_

Reply:
Ahh, good news. hopped over to scikit-learn and noticed they had found a way to include a link option in the "Issue Select" page. They configure this here: https://github.com/scikit-learn/scikit-learn/blob/master/.github/ISSUE_TEMPLATE/config.yml

@jcohenadad we can also use this for SCT, etc. to redirect users to the spinalcordmri forum.
![Screenshot from 2020-12-23 10-34-22](https://user-images.githubusercontent.com/16181459/103013549-59e3f300-450b-11eb-936d-54375717f349.png)

Reply:
this is *awesome* 🙌
thank you for digging @joshuacwnewton

Reply:
I'm working on a new commit that tries to address "everything" in one go; I'll push to this branch for review today